cmd/influx/config/config.go

```diff
@@ -14,78 +14,144 @@ import (
 
 // Config store the crendentials of influxdb host and token.
 type Config struct {
+	Name string `toml:"-" json:"-"`
 	Host string `toml:"url" json:"url"`
 	// Token is base64 encoded sequence.
-	Token  string `toml:"token" json:"token"`
-	Org    string `toml:"org" json:"org"`
-	Active bool   `toml:"active" json:"active"`
+	Token          string `toml:"token" json:"token"`
+	Org            string `toml:"org" json:"org"`
+	Active         bool   `toml:"active,omitempty" json:"active,omitempty"`
+	PreviousActive bool   `toml:"previous,omitempty" json:"previous,omitempty"`
 }
 
 // DefaultConfig is default config without token
 var DefaultConfig = Config{
-	Host:   "http://localhost:9999",
-	Active: true,
+	Name: "default",
+	Host: "http://localhost:9999",
 }
 
 // Configs is map of configs indexed by name.
 type Configs map[string]Config
 
 // ConfigsService is the service to list and write configs.
 type ConfigsService interface {
-	WriteConfigs(pp Configs) error
-	ParseConfigs() (Configs, error)
+	CreateConfig(Config) (Config, error)
+	DeleteConfig(name string) (Config, error)
+	UpdateConfig(Config) (Config, error)
+	SwitchConfig(name string) (Config, error)
```

Comment:
can we name this something better? still feels ambiguous to me. something `SetActive` so its more obvious maybe?

Reply:
what do you think about SwitchActive ?

Reply:
`SetActive`?